sdks/python/setup.py

```diff
@@ -17,29 +17,50 @@
 
 """Apache Beam SDK setup configuration."""
 
-import os
+
 import platform
-import re
 import setuptools
+import re
 
 
-# Currently all compiled modules are optional  (for performance only).
+# Currently all compiled modules are optional (for performance only).
 if platform.system() == 'Windows':
   # Windows doesn't always provide int64_t.
   cythonize = lambda *args, **kwargs: []
 else:
   try:
-    # pylint: disable=g-statement-before-imports,g-import-not-at-top
-    from Cython.Build import cythonize
+      # pylint: disable=g-statement-before-imports,g-import-not-at-top
```

Comment:
2 spaces please instead of 4. I noticed that our linter is skipping the setup.py file. I will fix that.